### DISCUSSION TITLE
height as a variable doesn't work

### DISCUSSION THREAD

**Discussion by wallyhpipp** [tinymce/tinymce-react#378] [Q&A] [answered]
I'm using tinymce in a Next js function component, and I'm sending it three props. The props for height doesn't has no effect and gives me an error in the console: Invalid value passed for the height option. My code:

<Editor
        id="YOUR_FIXED_ID"
        onInit={(evt, editor) => (editorRef.current = editor)}
        initialValue={initialValue} //props variable
        onEditorChange={onChange} //props variable
        init={{
          theme_advanced_resizing: true,
          theme_advanced_resizing_use_cookie: false,
          //   height: { height }, //props variable -- doesn't work
          height: 330, //works
          menubar: false,
          toolbar:
            "undo redo | formatselect | " +
            "bold italic backcolor | alignleft aligncenter " +
            "alignright alignjustify | bullist numlist outdent indent | " +
            "removeformat | help",
          content_style:
            "body { font-family:Helvetica,Arial,sans-serif; font-size:14px }",
        }}
      />

Thanks,
Wally

**Comment by tiny-james** [comment, marked as answer]
When you say:
```jsx
<Editor
init={{
// ... removed lines
height: { height },
// ... removed lines
}}
/>
```
You are assigning an object containing the property height instead of a number. You need to remove the brackets around it because you are already in a Javascript context.

**Comment by wallyhpipp** [comment]
Thanks for the fix and explaination, James.

Cheers,
Wally
